### PR TITLE
ALI token added (mainnet/base-mainnet)

### PR DESCRIPTION
**Pull request by vgorin**
**Description**

ALI token bridging request for the base mainnet: Ethereum mainnet <-> base mainnet

**Tests**

no tests, this is an already deployed token addition only

**Additional context**

This PR continues (and finishes) the already merged PR #555 which successfully bridged the ALI token in base testnet <-> goerli testnet

**Metadata**

- Completes https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/555

### CODE REVIEW

**Comment by vgorin** [comment]
Hello @ethereum-optimism/devxpod, trying to follow up here so the PR doesn't get automatically closed.

File: data/ALI/data.json
Comment (hamdiallam) [inline data/ALI/data.json:15]:
```suggestion
    "base": {
```

**Comment by roberto-bayardo** [comment]
taking a look

**Comment by roberto-bayardo** [comment]
Need one more sign-off before I can approve (slow going due to people being out of office), but we're on it.

**Review by roninjin10 — APPROVED** [review]

**Review by roberto-bayardo — DISMISSED** [review]
good to go!

**Review by roninjin10 — APPROVED** [review]

**Comment by vgorin** [comment]
Hello, guys, we need some help to resolve this mergify update problem.

Our fork is inside the organization and there is a problem with granting the mergify bot update permissions on the repo. But we can add individual contributors.

If you can share the github ID of the user who installed the mergify bot, then we can add this user to the repo and hopefully the mergify actions succeed.

At the moment I've updated our fork manually.

**Comment by roberto-bayardo** [comment]
removing stale label will check back in on this one next week

**Comment by roberto-bayardo** [comment]
Sorry appears I already approved this, good to merge @hamdiallam

**Review by roberto-bayardo — APPROVED** [review]

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**
